.goreleaser.yml

```diff
@@ -0,0 +1,98 @@
+project_name: migrate
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm
+      - arm64
+      - 386
+    goarm:
+      - 7
+    main: ./cmd/migrate
+    ldflags:
+      - '-w -s -X main.Version={{ .Version }} -extldflags "static"'
+    flags:
+      - "-tags={{ .Env.DATABASE }} {{ .Env.SOURCE }}"
+      - "-trimpath"
+    ignore:
+      - goos: windows
+        goarch: arm
+nfpms:
+  - homepage: "https://github.com/golang-migrate/migrate"
+    maintainer: "dhui@users.noreply.github.com"
+    license: MIT
+    description: "Database migrations"
+    formats:
+      - deb
+    file_name_template: "{{ .ProjectName }}.{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
+dockers:
```

Comment:
Is there a way to simplify this? e.g. use a matrix of os and arch to ship docker images for?

Reply:
It's probably possible and I'm happy to work on it later on. But could take some time to test and develop and for me, with the time I have, that would be next week at best.

Reply:
This isn't a blocker for this PR since there's only 2 elements in the list for now. Once we get to 3 or 4, we should move to a matrix version if it's possible. Looking at the docs and code, it doesn't seem like it is...